recipes/mobie/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "mobie" %}
+{% set version = "3.0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5b418e4c190e07e0c9761e25a396fc7cea1f75bd59010682386bffbe161e61d8
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - mobie=mobie:launch_mobie
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - jgo >=0.4.0
+    - pip
+    - python
+  run:
+    - jgo >=0.4.0
+    - python >=3.7
+    - openjdk =8
+    - maven
+
+test:
+  imports:
+    - mobie
+  commands:
+    - mobie --help
+
+about:
+  home: "https://github.com/mobie/mobie-viewer-fiji"
```

The package metadata was wrong, this is the correct URL:
```suggestion
  home: "https://github.com/mobie/mobie-cmd"
```